test/extended/security/supplemental_groups.go

```diff
@@ -33,7 +36,18 @@ var _ = g.Describe("[security] supplemental groups", func() {
 			fsGroup := int64(1111)
 			supGroup := int64(2222)
 
-			_, err := oc.AsAdmin().Run("adm").Args("policy", "add-scc-to-user", "anyuid", oc.Username()).Output()
+			err := retry.RetryOnConflict(retry.DefaultRetry, func() error {
+				_, err := oc.AsAdmin().Run("adm").Args("policy", "add-scc-to-user", "anyuid", oc.Username()).Output()
+				if err == nil {
+					return nil
+				}
+				if exitErr, ok := err.(*exutil.ExitError); ok {
+					if strings.HasPrefix(exitErr.StdErr, "Error from server (Conflict):") {
+						return errors.NewConflict(schema.GroupResource{}, "", err)
```

Comment:
Please add a comment about why this is necessary.

Reply:
Added in db68ca7637bbcea23f13d2ba2ab558ca37926de6.